resource-managers/mesos/pom.xml

```diff
@@ -74,6 +74,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Is this really needed?

I don't see you adding specific tests for this, so wonder why you need the explicit dependency when other modules that depend on `spark-core` don't.